test/release/examples/primers/domains.chpl

```diff
@@ -30,49 +30,53 @@
 config var n = 10;
 
 var RD: domain(3) = {1..n, 1..n, 1..n};
-writeln(RD);
+writeln(RD); // {1..10, 1..10, 1..10}
 
 //
 // Rectangular domains have a set of methods that enable convenient
 // reuse of existing domains.
 //
 // The :proc:`~ChapelDomain.expand` method returns a new domain that is expanded or
-// contracted depending on the sign of the offset argument.
+// contracted in each dimension depending on the value and sign of the offset
+// argument.  The expansion/contraction in each dimension happens symmetrically to
+// the lower and upper bound.
 //
-var RDbigger = RD.expand((1,1,1));
-writeln(RDbigger);
-var RDsmaller = RD.expand((-1,-1,-1));
-writeln(RDsmaller);
+var RDexp = RD.expand((1,2,-2));
+writeln(RDexp); // {0..11, -1..12, 3..8}
 
 //
-// The :proc:`~ChapelDomain.exterior` method returns a new domain that is the
-// exterior portion of the current domain.  A positive offset specifies that
-// the exterior should be taken from the high bound; a negative offset, the low
-// bound.
+// The :proc:`~ChapelDomain.exterior` method returns a new domain that is an
+// exterior portion of the current domain.  This is not the "border" of the
+// domain (which is not a rectangular domain), in particular the new domain
+// will share no indices with the original domain unless an offset is 0, in which
+// case the respective range will be that of the original domain.
+// Inputs specify the width of the new domain in the respective dimension.
+// A positive value specifies that the exterior should be taken from the high
+// bound; a negative offset, the low bound.
 //
-var RDext_p = RD.exterior((1,1,1));
-writeln(RDext_p);
-var RDext_n = RD.exterior((-1,-1,-1));
-writeln(RDext_n);
+var RDext0 = RD.exterior((1,4,-4));
+writeln(RDext0); // {11..11, 11..14, -3..0}
+var RDext1 = RD.exterior((0,4,0));
+writeln(RDext1); // {1..10, 11..14, 1..10}
```

Comment:
Somehow I think that 1-based indexing will be clearer: `RDext0` --> `RDext1` and `RDext1` --> `RDext2`. The `0` in `RDext0` seems to communicate something special about that variable. This may be misleading because there is nothing 0-related in that variable. Ditto `RDtrans0` and `RDtrans1` below.

Reply:
Done; this now agrees with the convention used in subdomains section.